packages/terra-application-header-layout/src/terra-dev-site/doc/example/HeaderWireframe.jsx

```diff
@@ -48,10 +48,10 @@ const HeaderWireframe = () => {
     <div className={cx('content-wrapper')}>
       <div className={cx('responsive-element-wrapper')}>
         <ResponsiveElement
-          defaultElement={tinyHeader}
           tiny={tinyHeader}
-          small={smallHeader}
-          medium={mediumHeader}
+          small={tinyHeader}
+          medium={smallHeader}
+          large={mediumHeader}
```

Comment:
Do we want to rename these elements to match the new breakpoints? Probably not a big deal either way

Reply:
Updated: https://github.com/cerner/terra-framework/pull/745/commits/d667a6c9214d2ba6c1d71e62da258d77776a9cb7